### ISSUE TITLE
Optimization for Raspberry Pi

### ISSUE THREAD

**Issue by simonszu** [hjacobs/kube-ops-view#170]
We have a Raspberry Pi 3 hooked up to a big screen for displaying stats. The Pi starts an X server which just contains a single instance of Chromium 60. This is fine for Grafana. 

But apparently kube-ops-view seems to be so resource-heavy that the Pi isn't able to display it correctly. It just displays the black background and 4 big grey rectangles in the corners of the browser window. After that, nothing happens, the mouse input begins to lag, and a window pops up that the browser tab isn't responding. Further waiting results in no progress. 

So i am opening this issue, since i think that we are not the only team which has this setup of raspberry pi plus big screen, and maybe kube-ops-view needs some optimizations toward tiny hardware setups.

**Comment by hjacobs** [comment]
@simonszu did you try the `renderer` UI option? See the [kube-ops-view documentation](http://kubernetes-operational-view.readthedocs.io/en/latest/ui-options.html). You can disable WebGL via this flag --- this might be better for weak hardware.

**Comment by simonszu** [comment]
Yes, this is a step towards a working setup. But unfortunately, this still spawns high load on the Raspi, and causes a full freeze after some time, prepended by mouse lag, but after the freeze, even the mouse isn't working. Maybe an option to disable some of the animations would be cool - at least the "new pod" animation is already quite laggy.

**Comment by towolf** [comment]
Just a minor data point, we are using "Webkit for Wayland" (instead of Chromium) by building a custom version [of this](https://github.com/WebPlatformForEmbedded/buildroot) and it is really smoothly animating the WebGL. It has no problems at all on an Rpi 3.
